src/legacy/core_plugins/kbn_vislib_vis_types/public/types.ts

```diff
@@ -59,7 +59,7 @@ export interface Style {
 }
 
 export interface Scale {
-  boundsMargin?: number | '';
+  boundsMargin?: number;
```

Comment:
I think the reason for the empty string type in this place was that it was a valid value before and saved objects might still contain it. @timroes is that right? If that's the case we should keep it and probably add a comment

Reply:
Yeah I think we can't remove them now, since the way the editor behaves so far adds emptry string in case the user leaves it empty. Not a good solution but unfortunately we're now stuck with that for some time in the saved objects. @sulemanof please correct me if I am wrong, and the issues we talked about the past days don't apply here?

Reply:
I reverted `''` type since we can have saved objects that already contain `'' `.